woocommerce/woocommerce
# add: return to lys from cys

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #47604.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Set up WooCommerce core, do not complete customise your store task
2. Proceed to "Launch your Store" task
3. Follow the "Customise Store" task item from within "Launch your Store" interface
4. Follow through with the customise store task
5. You should not see the transitional page (the one with a survey) and should be redirected back to launch your store after saving customisations.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
